core_backend/app/users/models.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime
+from typing import Optional
+
+from sqlalchemy import (
+    DateTime,
+    String,
+    # delete,
+    select,
+)
+from sqlalchemy.exc import NoResultFound
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import Mapped, mapped_column
+
+from ..models import Base
+from .schemas import UserCreate
+
+
+class UserDB(Base):
+    """
+    SQL Alchemy data model for users
+    """
+
+    __tablename__ = "user"
+
+    user_id: Mapped[str] = mapped_column(String, primary_key=True, nullable=False)
```

Comment:
Noticed no `access_level`. I'm cool with it. 
Though would we want a super_user that has access to content?

Reply:
I think at some point we'll want access_level and super_user, but I'm trying to keep these PRs lean. Will make a ticket for adding a super_user (since we'll definitely need that at some stage).